Update directx-sdk--august-2009-.md

walbourn opened the pull request:
Microsoft.DXSDK.D3DX is a NuGet package option for legacy D3DX

walbourn committed:
Update directx-sdk--august-2009-.md

Microsoft.DXSDK.D3DX is a NuGet package option for legacy D3DX

PRMerger14 commented:
@walbourn : Thanks for your contribution! The author(s) have been notified to review your proposed change.

stevewhims committed:
Dereferencing aka.ms urls

It seems that aka.ms and FWLINK urls are problematic for localization, so we expand them these days.

PRMerger19 commented:
@stevewhims : Thanks for your contribution! The author(s) have been notified to review your proposed change.
